about/changelog.md

```diff
@@ -8,6 +8,16 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+## :bug: Fixes and improvements
```

Comment:
## ⚡ Performance improvements for live migration to Timescale Cloud

- With updated table filtering, Timescale Cloud retries automatically if you have an error or backoff during live migration. 
- Improvements to pbcopydb increase performance for live migration and remove the possibility for deadlock. 

To learn more, see the [live migration docs](https://docs.timescale.com/migrate/latest/live-migration/).

Reply:
I think I wouldn't mention `deadlock and intermittent crashes`. What about instead we mention that a bug that stops live-migration from moving forward and causing it to display a warning message saying that the replay rate is 0 B/s was fixed?

Reply:
Or maybe it's too specific?

Reply:
Updated ☝️